Update section on embed ipywidgets (#181)
widgetti/ipyvuetify: docs/template_usage.rst

Embed ipywidgets
----------------

> Any ipywidget can be embedded by setting them in a trait and adding widget_serialization, accessing them in the template with the `jupyter-widget` tag:

Reviewer: ```suggestion
Any ipywidget can be embedded by setting them in a trait and adding widget_serialization, accessing them in the template with the ``jupyter-widget`` tag:
```

I think single quote are .md symbols, you need to use 2 of them in rst files.

Author: Thanks for the suggestion, it is committed.

            'content': slider2
        }]).tag(sync=True, **widgets.widget_serialization)

        single_widget = traitlets.Any().tag(sync=True, **widgets.widget_serialization)
    
        def __init__(self, **kwargs):

>             self.single_widget = widgets.IntSlider(description='Single slider', value=40)
>             self.single_widget.__init__(**kwargs)

Reviewer 2: Calling the constructor twice cannot be right, it probably has weird side effects (creating 2 comms).
```suggestion
            self.single_widget = widgets.IntSlider(description='Single slider', value=40, **kwargs)
```

Author: I agree, but the suggested change does not work when calling MyComponent(value=20), i.e. for arguments already defined.
```suggestion
            self.single_widget = widgets.IntSlider(description='Single slider', value=40)
            for k, v in kwargs.items():
                setattr(self.single_widget, k, v)
```
Alternatively, the default arguments are passed to `__init__`:
```
def __init__(self, description='Single slider', value=40, **kwargs):
    self.single_widget = widgets.IntSlider(
        description=description, 
        value=value, 
        **kwargs
    )
```
The second option may look more readable. Any preference?
